common-npm-packages/azure-arm-rest/azCliUtility.ts

```diff
@@ -188,4 +189,49 @@ function isAzVersionGreaterOrEqual(azVersionResultOutput: string, versionToCompa
         tl.error(`Error checking Azure CLI version: ${error.message}`);
         return false;
     }
+}
+
+async function getAzModuleMajorReleases(moduleName: string): Promise<string> {
```

Comment:
Why it's named getAzModuleMajorReleases when we return a tag for a single release?

Reply:
Changed the function name as getLatestAzureModuleReleaseVersion